docs/Plugins.md

```diff
@@ -21,12 +21,12 @@ The [Built-in Plugins](#built-in-plugins) are compiled and installed when you se
 
 #### Plugin Object
 
-| Key     | Required | Default Value | Type                 | Description                                                  |
-| ------- | :------: | ------------- | -------------------- | ------------------------------------------------------------ |
-| library |    ✓     |               | string               | the name of the library that contains the plugin.            |
+| Key     | Required | Default Value | Type                 | Description                                                                                                                 |
+| ------- | :------: | ------------- | -------------------- | --------------------------------------------------------------------------------------------------------------------------- |
+| library |    ✓     |               | string               | the name of the library that contains the plugin.                                                                           |
 | name    |    ✓     |               | string               | the name of the plugin. This name is used to find the `<name>_plugin_new` method that creates a new instance of the plugin. |
-| enabled |          | true          | **true** / **false** | control whether a configured plugin is enabled or disabled   |
-|         |          |               |                      | *Additional elements can be added, they will be passed into the `parse_config` method of the plugin.* |
+| enabled |          | true          | **true** / **false** | control whether a configured plugin is enabled or disabled                                                                  |
+|         |          |               |                      | *Additional elements can be added, they will be passed into the `parse_config` method of the plugin.*                       |
 
 
 ## Built-in Plugins
```

Comment:
What are the distinguishing characteristics of a "built-in plugin" and a "user plugin"? Is there a promotion process?

Reply:
I would suggest filing tickets to move any plugin that does not meet the criteria for "built-in" to be a "user" plugin.

Reply:
I am using the language to be consistent with existing references in the code or repository to describe those plugins normally included in an installation. 

But at present, the only distinguishing characteristic of a "built-in" plugin is that it is shipped with trunk recorder for the benefit of the majority of users, and can only be maintained by filing a PR against the repo. 

As fas as current candidates to move, the the `streamer` plugin has likely been broken since at least 4.6 without anyone noticing, and `stat_socket` can probably benefit by bundling it into https://github.com/TrunkRecorder/trunk-recorder-status-server, instead of splitting the functionality in two halves.